string table
string key
---
string value
